compiler/src/iree/compiler/Codegen/SPIRV/ConvertToSPIRVPass.cpp

```diff
@@ -405,6 +405,18 @@ void ConvertToSPIRVPass::runOnOperation() {
     }
   }
 
+  /// Rewrite exti(bitcast) as a mix of vector.shuffle + bitwise arithmetic to
+  /// help manage sub-byte types.
```

Comment:
This sounds a bit vague, could you give an example that shows what it means to manage sub-byte types?

Reply:
Does the new comment here explain clearly? Or do you want explicit IR examples as well.